tools/PI/DevHome.PI/Helpers/ExternalTool_v1.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json.Serialization;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace DevHome.PI.Helpers;
+
+public enum ExternalToolArgType
+{
+    None,
+    ProcessId,
+    Hwnd,
+}
+
+public partial class ExternalTool_v1 : ObservableObject
+{
+    public string ID { get; private set; }
+
+    public string Name { get; private set; }
+
+    public string Executable { get; private set; }
+
+    [JsonConverter(typeof(EnumStringConverter<ExternalToolArgType>))]
+    public ExternalToolArgType ArgType { get; private set; } = ExternalToolArgType.None;
+
+    public string ArgPrefix
+    {
+        get; private set;
+    }
+
+    public string OtherArgs
```

Comment:
`OtherArgs` makes me think this class can hold two different args.  What is `OtherArgs` and why not just `args`?

Reply:
This is the legacy type definition. In the original implementation, we had pid/hwnd-specific args plus other args. This has been removed in the newer version. The legacy version is only used to enable migrating an old-format file to the new format.